lib/vim-plugin.nix

```
@@ -113,7 +113,7 @@
                 extraPlugins = extraPlugins ++ [
                   (cfg.packageDecorator cfg.package)
                 ];
-                globals = lib.mapAttrs' (n: lib.nameValuePair (globalPrefix + n)) (cfg.settings or { });
+                globals = lib.applyPrefixToAttrs globalPrefix (cfg.settings or { });
```

```suggestion
                globals = lib.nixvim.applyPrefixToAttrs globalPrefix (cfg.settings or { });
```

Or `lib.nixvim.utils.applyPrefixToAttrs` if we don't alias it to the top-level in `lib/default.nix`